packages/fxa-auth-server/scripts/activate-new-signing-key.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
```

Comment:
@jrgm and @jbuck will need to update their signing key scripts to point to the new location.

Reply:
It's also in still in the old location for now.